python/llm/example/GPU/vLLM-Serving/README.md

```diff
@@ -17,7 +17,7 @@ In this example, we will run Llama2-7b model using Arc A770 and provide `OpenAI-
 
 ### 0. Environment
 
-To use Intel GPUs for deep-learning tasks, you should install the XPU driver and the oneAPI Base Toolkit 2024.0. Please check the requirements at [here](https://github.com/intel-analytics/ipex-llm/tree/main/python/llm/example/GPU#requirements).
+To use Intel GPUs for deep-learning tasks, you should install the XPU driver and the oneAPI Base Toolkit 2024.1. Please check the requirements at [here](https://github.com/intel-analytics/ipex-llm/tree/main/python/llm/example/GPU#requirements).
```

Comment:
Is the link correspond with oneapi 2024.1? ipex-llm is mainly using 2024.0 on arc?

Reply:
Fixed using link:https://www.intel.com/content/www/us/en/docs/oneapi/installation-guide-linux/2024-1/overview.html